src/common/buildInfo.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// tslint:disable-next-line:no-var-requires
+const buildConfig = require("../buildConfig.json");
```

Comment:
Let's call it something like appConfig, as it applies at runtime instead of build time.

Reply:
Also rename buildConfig.json